From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Evan Maddock <maddock.evan@vivaldi.net>
Date: Fri, 16 Jan 2026 15:33:39 -0500
Subject: [PATCH] Don't auto-detect shredder

This fixes the issue where the engine list will accidentally match
against the Shredder utility from `rmlint` if installed. If this
happens, pychess then tries to start the utility, causing another window
to appear.

Signed-off-by: Evan Maddock <maddock.evan@vivaldi.net>
---
 lib/pychess/Players/engineList.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/pychess/Players/engineList.py b/lib/pychess/Players/engineList.py
index 71928af7..8e451548 100644
--- a/lib/pychess/Players/engineList.py
+++ b/lib/pychess/Players/engineList.py
@@ -72,7 +72,7 @@ ENGINES_LIST = [
     ENGINE("fizbo", "uci", "us", 3346, AUTO_DETECT, None),
     ENGINE("andscacs", "uci", "ad", 3337, AUTO_DETECT, None),
     ENGINE("booot", "uci", "ua", 3357, AUTO_DETECT, None),  # Formerly XB
-    ENGINE("shredder", "uci", "de", 3323, AUTO_DETECT, None),
+    ENGINE("shredder", "uci", "de", 3323, NO_AUTO_DETECT, None),
     ENGINE("schooner", "xboard", "ca", 3288, AUTO_DETECT, None),
     ENGINE("arasan", "uci", "us", 3274, AUTO_DETECT, None),
     ENGINE("rubichess", "uci", "de", 3292, AUTO_DETECT, None),
